osmnx/geometries.py

```diff
@@ -615,36 +618,46 @@ def _closed_way_is_linestring_or_polygon(element, polygon_features):
             is_polygon = False
```

Comment:
@AtelierLibre this is already `False` by default 10 lines above... can we just `pass` here instead?

Reply:
Yes, I think so.

Reply:
Resolved in 8ce1dc1